### ISSUE TITLE
Nova 4 support

### ISSUE THREAD

**Issue by stepanenko3** [chaseconey/nova-external-image#12]

**Comment by tobyberesford** [comment]
Is this library meant to be able to work in Nova 4 @chaseconey ? it just displays a blank space for me instead of the image? How much work is it to fix it? Thanks!

**Comment by jeanfrancoisgb** [comment]
I really would need Nova 4 support as well.

**Comment by tooshay** [comment]
Has anyone found an alternative with Nova 4 support by any chance? @jeanfrancoisgb? @tobyberesford?

**Comment by jeanfrancoisgb** [comment]
@tooshay Not ideal but I ended up using the Avatar field with custom URL.


```
Avatar::make('Main Image')->disk('public')
->disableDownload()
->thumbnail(function ($value, $disk) {
    return $this->getUrl(false);
})
->preview(function ($value, $disk) {
    return $this->getUrl(false);
})
->showOnIndex()
->squared()
->deletable(false),
```
instead of getUrl, just return the exteral URL of the image.

**Comment by clementmas** [comment]
There's a PR that looks good: https://github.com/chaseconey/nova-external-image/pull/13.

@chaseconey, can it be merged?

**Comment by andrzejkupczyk** [comment]
You are not limited to using only the `Avatar` field. You can also use any other field that extends the `File` field. 

```php
Image::make('Image')
    ->preview(fn () => $this->externalImageUrl)
    ->thumbnail(fn () => $this->externalImageUrl)
```

Consult the `Laravel\Nova\Fields\File` class and [documentation](https://nova.laravel.com/docs/resources/file-fields.html#customization) for more information.

**Comment by chaseconey** [comment]
Fixed in #13
